bin/rbenv-ctags

```diff
@@ -3,8 +3,8 @@
 # Summary: Generate ctags for a given version's standard library
 #
 # Usage: rbenv ctags
-#        rbenv ctags <version> [<version> ...]
-#        rbenv ctags --all
+#        rbenv ctags <version> [<version> ...] [--ripper]
+#        rbenv ctags --all [--ripper]
```

Comment:
Let's go with:

```bash
# Usage: rbenv ctags [--ripper]
#        rbenv ctags [--ripper] <version> [<version> ...]
#        rbenv ctags [--ripper] --all
```

The option is valid for all 3 variants. And putting it up front makes the parallel obvious.